Tests/GRPCTests/ConnectionManagerTests.swift

```diff
@@ -491,15 +520,16 @@ extension ConnectionManagerTests {
       return next
     }
 
-    let readyChannelMux: EventLoopFuture<HTTP2StreamMultiplexer> = self.waitForStateChanges([
-      Change(from: .idle, to: .connecting),
-      Change(from: .connecting, to: .transientFailure),
-    ]) {
-      // Get a HTTP/2 stream multiplexer.
-      let readyChannelMux = manager.getHTTP2Multiplexer()
-      self.loop.run()
-      return readyChannelMux
-    }
+    let readyChannelMux: EventLoopFuture<NIOHTTP2Handler.StreamMultiplexer> = self
+      .waitForStateChanges([
+        Change(from: .idle, to: .connecting),
+        Change(from: .connecting, to: .transientFailure),
+      ]) {
+        // Get a HTTP/2 stream multiplexer.
+        let readyChannelMux = manager.getHTTP2Multiplexer()
+        self.loop.run()
+        return readyChannelMux
+      }
```

Comment:
The formatter did a really bad job here...

```suggestion
    let readyChannelMux = self.waitForStateChanges([
        Change(from: .idle, to: .connecting),
        Change(from: .connecting, to: .transientFailure),
    ]) { () -> EventLoopFuture<NIOHTTP2Handler.StreamMultiplexer> in
      // Get a HTTP/2 stream multiplexer.
      let readyChannelMux = manager.getHTTP2Multiplexer()
      self.loop.run()
      return readyChannelMux
    }
```